config.json

```diff
@@ -1117,6 +1117,19 @@
       "unlocked_by": "triangle",
       "uuid": "52d11278-0d65-4b5b-b387-1374fced3243"
     },
+    {
+      "core": false,
+      "difficulty": 8,
+      "slug": "rational-numbers",
+      "topics": [
+        "floating_point_numbers",
+        "mathematics",
+        "classes",
```

Comment:
These look like good topics 🙂 But I'm not sure about `floating_point_numbers` and `classes`, how are they relevant to this exercise?

Reply:
@FridaTveit Well, this exercise is similar to `complex-numbers`, and that exercise has those topics. Floating-point numbers (doubles) are involved in some of the methods of this exercise, and this exercise does require that one create a class with some data hiding. But if you don't think this exercise warrants those topics, I'll remove them.

Reply:
I think `classes` has recently been removed as a topic of `complex-numbers` :slightly_smiling_face: